static/src/javascripts/projects/commercial/modules/dfp/lazy-load.spec.ts

```diff
@@ -1,9 +1,17 @@
+import { isInVariantSynchronous } from 'common/modules/experiments/ab';
 import type { Advert } from './Advert';
 import { dfpEnv } from './dfp-env';
 import { getAdvertById as getAdvertById_ } from './get-advert-by-id';
 import { enableLazyLoad } from './lazy-load';
 import { loadAdvert } from './load-advert';
 
+jest.mock('lodash-es', () => ({
+	...jest.requireActual('lodash-es'),
+	// Mock `once` as the identity function so we can re-run `enableLazyLoad`
+	// and generate different intersection observers
+	once: jest.fn().mockImplementation(<T>(f: T) => f),
+}));
+
```

Comment:
Not this mock but some of the mocks below look like they don't need to pass the second parameter to stub the module. Might be worth cleaning up while we're here.